modules/indexer/stats/db.go

```diff
@@ -38,8 +38,8 @@ func (db *DBIndexer) Index(id int64) error {
 	// Get latest commit for default branch
 	commitID, err := gitRepo.GetBranchCommitID(repo.DefaultBranch)
 	if err != nil {
-		log.Error("Unable to get commit ID for defaultbranch %s in %s", repo.DefaultBranch, repo.RepoPath())
-		return err
+		log.Debug("Unable to get commit ID for defaultbranch %s in %s ... skipping this repository", repo.DefaultBranch, repo.RepoPath())
```

Comment:
We should do this according if the error type is `ErrBranchIsNotExist`.

Reply:
I'm not sure because really the error is not very helpful - but I've done as you requested.